x-pack/plugins/infra/public/pages/metrics/layouts/container.ts

```diff
@@ -82,46 +120,109 @@ export const containerLayoutCreator: InfraMetricLayoutCreator = theme => [
       },
       {
         id: InfraMetric.containerNetworkTraffic,
-        label: 'Network Traffic',
+        label: i18n.translate(
+          'xpack.infra.metricDetailPage.containerMetricsLayout.networkTrafficSection.sectionLabel',
+          {
+            defaultMessage: 'Network Traffic',
+          }
+        ),
         requires: ['docker.network'],
         type: InfraMetricLayoutSectionType.chart,
         visConfig: {
           formatter: InfraFormatterType.bits,
           formatterTemplate: '{{value}}/s',
           type: InfraMetricLayoutVisualizationType.area,
           seriesOverrides: {
-            rx: { color: theme.eui.euiColorVis1, name: 'in' },
-            tx: { color: theme.eui.euiColorVis2, name: 'out' },
+            rx: {
+              color: theme.eui.euiColorVis1,
+              name: i18n.translate(
+                'xpack.infra.metricDetailPage.containerMetricsLayout.networkTrafficSection.networkRxRateSeriesLabel',
+                {
+                  defaultMessage: 'in',
+                }
+              ),
+            },
+            tx: {
+              color: theme.eui.euiColorVis2,
+              name: i18n.translate(
+                'xpack.infra.metricDetailPage.containerMetricsLayout.networkTrafficSection.networkTxRateSeriesLabel',
+                {
+                  defaultMessage: 'out',
+                }
+              ),
+            },
           },
         },
       },
       {
         id: InfraMetric.containerDiskIOOps,
-        label: 'Disk IO (Ops)',
+        label: i18n.translate(
+          'xpack.infra.metricDetailPage.containerMetricsLayout.diskIoOpsSection.sectionLabel',
+          {
+            defaultMessage: 'Disk IO (Ops)',
+          }
+        ),
         requires: ['docker.diskio'],
         type: InfraMetricLayoutSectionType.chart,
         visConfig: {
           formatter: InfraFormatterType.number,
           formatterTemplate: '{{value}}/s',
           type: InfraMetricLayoutVisualizationType.area,
           seriesOverrides: {
-            read: { color: theme.eui.euiColorVis1, name: 'reads' },
-            write: { color: theme.eui.euiColorVis2, name: 'writes' },
+            read: {
+              color: theme.eui.euiColorVis1,
+              name: i18n.translate(
+                'xpack.infra.metricDetailPage.containerMetricsLayout.diskIoOpsSection.readRateSeriesLabel',
+                {
+                  defaultMessage: 'reads',
+                }
+              ),
+            },
+            write: {
+              color: theme.eui.euiColorVis2,
+              name: i18n.translate(
+                'xpack.infra.metricDetailPage.containerMetricsLayout.diskIoOpsSection.writesRateSeriesLabel',
```

Comment:
some nitpicks :innocent: 

```suggestion
                'xpack.infra.metricDetailPage.containerMetricsLayout.diskIoOpsSection.writeRateSeriesLabel',
```